examples/solid/start-bare/src/routes/__root.tsx

```diff
@@ -1,24 +1,30 @@
-import { createRootRoute, Link, Outlet } from '@tanstack/solid-router'
+import {
+  HeadContent,
+  Link,
+  Outlet,
+  Scripts,
+  createRootRoute,
+} from '@tanstack/solid-router'
+
 import appCss from '~/styles/app.css?url'
-import * as Solid from 'solid-js'
-import { TanStackRouterDevtools } from '@tanstack/solid-router-devtools'
 
 export const Route = createRootRoute({
   head: () => ({
     links: [{ rel: 'stylesheet', href: appCss }],
   }),
-  component: RootDocument,
+  component: RootComponent,
 })
 
-function RootDocument({ children }: { children: Solid.JSX.Element }) {
+function RootComponent() {
   return (
     <>
+      <HeadContent />
       <div class="p-2 flex gap-2 text-lg">
         <Link to="/">Index</Link>
         <Link to="/about">About</Link>
       </div>
       <Outlet />
-      <TanStackRouterDevtools position="bottom-right" />
+      <Scripts />
```

Comment:
`Scripts` should be fine in the server entrypoint.

Reply:
The problem was the Scripts component needs to be in the "Router" context so that doesn't work outside just in the entry.